department-of-veterans-affairs/caseflow
# Correspondence intake spelling fix

Resolves [spelling mistake in correspondence intake]

# Description
The text in the correspondence Intake previously stated 
"Review any previously **_complete_** tasks by the mail team and add new tasks for either the mail package or for linked appeals, if any.

it has been updated so that it now states
"Review any previously **_completed_** tasks by the mail team and add new tasks for either the mail package or for linked appeals, if any."

## Testing Plan


1. Log in as BVAAABSHIRE
2. Navigate to http://localhost:3000/queue/correspondence/fe9173f0-38d2-42c9-af9e-2857375d542f/intake
3. notice the changed spelling.

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="1305" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/7aab0b52-d7e9-4872-8271-fb5343a5961f">
|
<img width="1255" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/7b089588-2c2c-4db0-bf7c-ff3d838b52f7">
